DatadogSessionReplay/Sources/Feature/Baggages.swift

```diff
@@ -38,3 +39,10 @@ internal struct RUMContext: Codable, Equatable {
     /// Current view related server time offset
     let viewServerTimeOffset: TimeInterval?
 }
+
+extension DatadogContext {
+    /// The value indicating if replay is being performed by Session Replay.
+    var hasReplay: Bool? {
+        try? baggages[RUMDependency.hasReplay]?.decode()
+    }
+}
```

Comment:
question/ Where do we use it? Having this defined in SR module looks odd - is there really a use case where SR needs to decode its own baggage?

Reply:
I had added it to access `hasReplay` from `WebViewRecordReceiver`, so I can ignore webviews' snapshots accordingly. Since I'm not taking care of it in this PR, I can remove it for now, but I might need to re-add it in the future. If that's the case, did you have another place in mind, that would be more appropriate?

Reply:
Got it 👍. No strong opinion on this, but perhaps better to introduce alongside with usage. I thought it's a leftover 🙂

Reply:
What do you mean by "perhaps better to introduce alongside with usage"?

Reply:
Yes, lets remove it if it's not used 👍

Reply:
Already done ✅